hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieActiveTimeline.java

```diff
@@ -304,7 +304,7 @@ public void deleteInstantFileIfExists(HoodieInstant instant) {
         if (result) {
           LOG.info("Removed instant " + instant);
         } else {
-          throw new HoodieIOException("Could not delete instant " + instant);
+          throw new HoodieIOException("Could not delete instant " + instant + " and path is " + commitFilePath);
```

Comment:
"Could not delete instant " + instant + " with path " + commitFilePath

Reply:
changed @danny0405